extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.resteasy.test;
+
+import org.hamcrest.Matchers;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusDevModeTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+
+public class NotFoundExceptionMapperTestCase {
```

Comment:
I don't see any source change that would trigger a hot reload in this class. Is there a point running a `QuarkusDevModeTest` without reloading?

Reply:
I'll let @loicmathieu  give a detail answer but I think it is for running under the `dev` profile.

Reply:
I just realized the `NotFoundExceptionMapper` PRs were submitted to fix an issue in dev mode, so you're right @machi1990! It makes sense to use `QuarkusDevModeTest` without reloading here :)

Reply:
Yes the exception mapper is only used in dev mode that what I use the QuarkusDevModeTest